docs/roman/datamodels/models.rst

```diff
@@ -129,7 +155,7 @@ model into a new one using the update method::
 History information
 -------------------
 
-Models contain a list of history records, accessed through the
+Models may contain a list of history records, accessed through the
 `history` attribute.  This is just an ordered list of strings --
```

Comment:
Is there a history attribute on these models?

Reply:
history is not added by default but is available via the utils. JWST only uses history for fits_generator and for wfs_combine. I couldn't decide if we should keep it or not. I can remove it until we make a group decision.